src/main/java/at/ac/tuwien/damap/rest/version/VersionDOMapper.java

```diff
@@ -7,7 +7,7 @@
 @UtilityClass
 public class VersionDOMapper {
 
-    public VersionDO mapEntityToDO(DmpVersion version, VersionDO versionDO) {
+    public static VersionDO mapEntityToDO(DmpVersion version, VersionDO versionDO) {
```

Comment:
@UtilityClass already makes all methods static, was this change to make it explicit?

Reply:
might have been a remainder of prior change. will revert